src/module-1/cutStrings/index.js

```diff
@@ -1 +1,8 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = [], strLength = 0) => {
+  if(strLength <= 0) {
+    const lengthArr = arr.map(str => str.length)
+    strLength = Math.min(...lengthArr)
+  }
+  return arr.map(str => str.substring(0, strLength))
+}
+cutStrings(["adfg", "bkfkdl", "cdfffk", "dgldssvv"], 4)
```

Comment:
You shouldn't pass string length. The idea of this task to receive array of strings of different length, find the shortest string and shorten all the other strings to the same length.